source/fetch.ts

```diff
@@ -94,6 +100,23 @@ namespace fetch {
         return response;
     }
 
+
+    /**
+     * Allows to wait synchronously on one or multiple promises
```

Comment:
hmm this exactly what this function should prevent - it should block until all promises have resoloved or one rejected. The important thing is, the caller of wait should not use `await` since this would require the caller to be async as well.

Reply:
the function is called during another promises resolve ... e.g.,
font face is loaded via async fetch - if the async fetch resolved, the font data is processed, during this, additional referenced resources will be loaded asynchronous. but i would like to block/delay the font face resolve until all resources are fetched as well. Thus, the font face resolve uses this wait.

Reply:
It's impossible to block without async on the calling side.